README.MD

```
@@ -347,14 +347,21 @@ Arguments
 
-Abbreviate number
-Returns the string representation of the abbreviated number
+Returns the string representation of the abbreviated number, rounded to two decimal places.
 
 Arguments
 |Name|Type|Required|Description|
 |--|--|--|--|
 |`number`|`number`|`true`|The number to be abbreviated|
 
+Examples
+* Numbers below 1,000 are displayed without abbreviation
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)